app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
```

Comment:
Make it inherit from `ApiController` once you add that controller in the other PR